platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/PlatformState.java

```diff
@@ -34,16 +34,13 @@
 
 /**
  * State managed and used by the platform.
+ * @deprecated Use {@link PlatformStateAccessorSingleton}. This object should be accessed via the State API.
  */
-public class PlatformState extends PartialMerkleLeaf implements MerkleLeaf {
+@Deprecated(since = "0.54.0", forRemoval = true)
```

Comment:
Are there plans to change existing non-Hedera apps to not use this class? If not, then deprecation doesn't make sense.